server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -763,7 +765,12 @@ public Owner updateOwnerContentAccess(Owner owner, String updatedList, String up
             owner = this.ownerCurator.merge(owner);
             ownerCurator.flush();
 
-            this.refreshOwnerForContentAccess(owner);
+            this.syncOwnerLastContentUpdate(owner);
+            // TODO cleanup
+            // Delete cert when turning off SCA
+            if (currentMode.equals("org_environment") && updatedMode.equals("entitlement")) {
```

Comment:
We should be using the enums here rather than the strings.

Reply:
Done.